companies/google.md

```diff
@@ -50,3 +50,14 @@ additional_resources:
   - url: https://outline.com/8emaNG
     display_name: After a Twitter thread exposed the mistreatment of Black employees at Google, I ended my company's partnership to connect HBCU students with the tech giant. Here's why we decided to pull the plug.
 ---
+
```

Comment:
This looks like a carryover from your Google PR.  Could you merge `main` into your branch to correct it so this file doesn't show up in the PR?